android-app/app/src/main/assets/langs/english.toml

```diff
@@ -0,0 +1,71 @@
+[aliases]
```

Comment:
As a followup we'll want to load this file out of `extension/background/language/langs/english.toml`